.github/workflows/build_packages.yaml

```diff
@@ -453,7 +453,14 @@ jobs:
         build_machine:
           - aws-arm64
           - ubuntu-20.04
+        registry:
```

Comment:
This will make same image get built twice?
I think we could just build once and then push twice with different tags.

Reply:
Updated